app/src/main/java/com/github/se/cyrcle/model/report/ReportedObjectViewModel.kt

```diff
@@ -10,6 +11,8 @@ sealed class Report {
   data class Parking(val parkingReport: ParkingReport) : Report()
 
   data class Review(val reviewReport: ReviewReport) : Report()
+
+  data class Image(val imageReport: ImageReport) : Report()
```

Comment:
Maybe you could add documentation here just to explain how the argument works, the name being meaningful enough